Description: Connect to Freenode IRC by default for Ubuntu channels

=== modified file 'libempathy-gtk/empathy-irc-network-chooser.c'
Index: empathy-3.10.0/telepathy-account-widgets/tp-account-widgets/tpaw-irc-network-chooser.c
===================================================================
--- empathy-3.10.0.orig/telepathy-account-widgets/tp-account-widgets/tpaw-irc-network-chooser.c
+++ empathy-3.10.0/telepathy-account-widgets/tp-account-widgets/tpaw-irc-network-chooser.c
@@ -29,7 +29,7 @@
 #define DEBUG_FLAG TPAW_DEBUG_ACCOUNT | TPAW_DEBUG_IRC
 #include "tpaw-debug.h"
 
-#define DEFAULT_IRC_NETWORK "irc.gimp.org"
+#define DEFAULT_IRC_NETWORK "chat.freenode.net"
 #define DEFAULT_IRC_PORT 6667
 #define DEFAULT_IRC_SSL FALSE
 
